### PR TITLE
Api 7 5

### PR DESCRIPTION
**Pull request by ekansa**
### brief description of changes
I reviewed the current stable release of the API (https://github.com/archesproject/arches/blob/stable/7.5.3/arches/app/views/api.py) and the resource.py (https://github.com/archesproject/arches/blob/stable/7.5.3/arches/app/views/resource.py) source code to see if I could see where we needed to update the API documentation. 

Hopefully I caught everything!

#### issues addressed
See: (https://github.com/archesproject/arches-docs/issues/430)

#### further comments
I didn't look closely at the output examples to see if they also needed updates. I'd need an Arches instance with some public data to check that more completely.

---

This box **must** be checked
- [X] the PR branch was originally made from the base branch

This box **should** be checked
- [X] after these changes the docs build locally without error

This box **should only** be checked you intend to follow through on it (we can do it on our end too)
- [ ] I will `cherry-pick` all commits in this PR into other branches that should have them _after_ this PR is merged

### CODE REVIEW

**Comment by ekansa** [comment]
@robgaston ping! Hopefully this update accurately reflects the current state of the API?

File: docs/developing/reference/api.rst [hunk 1]
```diff
@@ -867,7 +868,7 @@ GeoJSON
     WARNING: including primary names has a big impact on performance and is best defered to an additional request
 
     :query resourceid: optional comma delimited list of resource instance UUIDs to filter feature data on
-    :query nodeid: optional node UUID to filter feature data on
+    :query nodeid: optional comma delimited list of node UUIDs to filter feature data on
```
Comment (robgaston) [inline docs/developing/reference/api.rst:871]: the parameter name here should actually be `nodeids`.  `nodeid` still works (only for a single node id) but should be considered deprecated
```suggestion
    :query nodeids: optional comma delimited list of node UUIDs to filter feature data on
```

File: docs/developing/reference/api.rst [hunk 2]
```diff
@@ -505,7 +506,7 @@ Resources
 
     Updates a single resource instance
 
-    :query format: {"json-ld", "arches-json"}
+    :query format: {"json-ld", "arches-json"} (default is ``json-ld``)
     :query indent: number of spaces to indent json output
```
Comment (robgaston) [inline docs/developing/reference/api.rst:510]: I  think that we can also indicate that the default for `indent` is `None` here
```suggestion
    :query indent: number of spaces to indent json output (default is ``None``)
```

**Review by robgaston — CHANGES_REQUESTED** [review]
a nice improvement! please see a few small suggestions for changes

**Comment by ekansa** [comment]
Thanks @robgaston ! I committed your suggestions. Thanks for the careful review and help!

**Review by robgaston — APPROVED** [review]
looks good!

**Pull request merged**